tests/testTriggers.go

```diff
@@ -6740,58 +6742,48 @@ func validateAutoFsTrim(contexts *[]*scheduler.Context, event *EventRecord) {
 					)
 					continue
 				}
-				log.Infof("Getting info : %s", v.ID)
+
+				log.Infof("Getting volume %s inspect response", v.ID)
 				appVol, err := Inst().V.InspectVolume(v.ID)
 				if err != nil {
-					log.Errorf("Error inspecting volume: %v", err)
+					UpdateOutcome(event, fmt.Errorf("error inspecting volume [%v]: %v", v.ID, err))
+					continue
 				}
-				attachedNode := appVol.AttachedOn
-
-				var fsTrimStatuses map[string]opsapi.FilesystemTrim_FilesystemTrimStatus
 
+				attachedNode := appVol.AttachedOn
 				t := func() (interface{}, bool, error) {
-					fsTrimStatuses, err = Inst().V.GetAutoFsTrimStatus(attachedNode)
+					fsTrimStatuses, err := Inst().V.GetAutoFsTrimStatus(attachedNode)
 					if err != nil {
 						return nil, true, fmt.Errorf("error autofstrim status node %v status", attachedNode)
 					}
-
-					return nil, false, nil
+					val, ok := fsTrimStatuses[appVol.Id]
+					var fsTrimStatus opsapi.FilesystemTrim_FilesystemTrimStatus
+					if !ok {
+						fsTrimStatus, _ = waitForFsTrimStatus(event, attachedNode, appVol.Id)
+					} else {
+						fsTrimStatus = val
+					}
+					log.Infof("autofstrim status for volume %v, status: %v", appVol.Id, val.String())
+					if fsTrimStatus != -1 {
+						if fsTrimStatus == opsapi.FilesystemTrim_FS_TRIM_COMPLETED {
+							return nil, false, nil
+						} else if fsTrimStatus == opsapi.FilesystemTrim_FS_TRIM_FAILED {
```

Comment:
Add Unknown also

Reply:
From the logs I can see Unknown is a temporary state but I am not sure

```bash
17:41:09  2024-05-16 12:11:01 +0000:[INFO] [{Longevity}] Autofstrim status for volume pvc-9134f7e0-59d4-4be1-be7b-1360d1ad4298, status: FS_TRIM_UNKNOWN

19:40:30  2024-05-16 14:10:28 +0000:[INFO] [{Longevity}] Autofstrim status for volume pvc-9134f7e0-59d4-4be1-be7b-1360d1ad4298, status: FS_TRIM_INPROGRESS
```